servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-queue-binder/pom.xml

```diff
@@ -13,7 +13,7 @@
   <artifactId>spring-cloud-azure-sample-stream-binder-servicebus-queue</artifactId>
   <groupId>com.azure.spring</groupId>
   <version>1.0.0</version>
-  <name>Spring Cloud Azure Sample Service Bus Queue Binder</name>
+  <name>Communicate to a Single Service Bus Queue via Spring Cloud Stream</name>
```

Comment:
To, A, Via

Reply:
This is not the rule here.

The principle(rule) here is 
1. keep Azure terminology, like Cosmos DB...
1. refer to https://capitalizemytitle.com/
2. better keep `with`, `and`, `to` the same case 
3. ...